internal/controllers/addon/controller.go

```diff
@@ -81,27 +82,35 @@ func NewAddonReconciler(
 		AddonOperatorNamespace: addonOperatorNamespace,
 		csvEventHandler:        csvEventHandler,
```

Comment:
Do additional dependencies which are used by the subreconcilers need to be members of the `AddonReconciler`? Specifically does the `AddonReconciler` need the `csvEventHandler`?

Reply:
> Specifically does the AddonReconciler need the csvEventHandler?

Yup: https://github.com/openshift/addon-operator/blob/main/internal/controllers/addon/controller.go#L198